components/modal/style.css

```diff
@@ -0,0 +1,49 @@
+.modal__content {
+  border: 1px solid var(--gray-light-secondary);
+  padding: 2rem;
+  margin-top: 2rem;
+}
+
+.modal {
+  border: 1px solid var(--gray-medium-primary);
+  max-width: 45rem;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 2rem;
+  border-radius: 5px;
+}
+
+.modal__title {
+  display: inline-block;
+}
+
+.modal__close {
+  background-color: transparent;
+  border: none;
+  font-size: inherit;
+  cursor: pointer;
+}
+
+.modal__header {
+  font-size: 2rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
can add utility class for this flex-row/flex-col, just suggestion

Reply:
Sure